installer/test/mix_helper.exs

```diff
@@ -11,24 +11,26 @@ defmodule MixHelper do
   end
 
   defp random_string(len) do
-    len |> :crypto.strong_rand_bytes() |> Base.encode64() |> binary_part(0, len)
+    len |> :crypto.strong_rand_bytes() |> Base.url_encode64() |> binary_part(0, len)
   end
 
   def in_tmp(which, function) do
```

Comment:
ExUnit now has a feature for creating tmp paths for tests, but this is fine for now. :)

Reply:
I actually wanted to use it, but ExUnit doesn't cleanup the tmp_dir afterwards, maybe that could be an (optional) improvement for ExUnit as well?

Reply:
We don’t clean up on purpose, to help debugging failed tests, but we should always clean up before the new run!